### PR TITLE
Added error response messages to the events controller.

### PR DESCRIPTION
**Pull request by XennMaestro**

### CODE REVIEW

File: app/Http/Controllers/Events.php [hunk 1]
```diff
@@ -26,7 +26,11 @@ public function index($id = null) {
      * @return Response
      */
     public function getEventByConferenceID($conferenceID) {
-    return Event::where('conferenceID',$conferenceID)->get();
+    $event = Event::find($conferenceID);
+      if (is_null($event)) {
+          return response("No events for conferenceID {$conferenceID}.", 404);
+      }
+      return Event::where('conferenceID',$conferenceID)->get();
```
Comment (brandonson) [inline app/Http/Controllers/Events.php]: I think this ends up doing two queries - should be able to just return `$event` and avoid the second one.

Also, consistent indentation please.

File: app/Http/Controllers/Events.php [hunk 2]
```diff
@@ -54,6 +58,10 @@ public function store(Request $request) {
      * @return Response
      */
     public function show($id) {
+      $event = Event::find($id);
+        if (is_null($event)) {
+            return response("No event for id {$id}.", 404);
+        }
         return Event::find($id);
```
Comment (brandonson) [inline app/Http/Controllers/Events.php]: Similar to above (double query + indentation)

**Comment by nikkimartin** [comment]
Any updates on this PR? It's been open for a while.

**Comment by XennMaestro** [comment]
Sorry got caught up with other subject i am making the changes now and will begin working on authentication and permissions as well.

**Comment by brandonson** [comment]
Looks good.

**Pull request merged**